relevanceai/apps/explorer_app/app.py

```diff
@@ -9,38 +11,37 @@ def preview_fields(self, fields: List = None, append=True):
         )
         self.document_card(fields=fields)
 
-    def document_card(self, 
-            fields:List=None, 
-            primary_field:str=None, 
-            secondary_field:str=None, 
-            image_field:str=None, 
-            layout_mode:str=None,
-            page_size:int=None,
-            append:bool=True
-        ):
+    def document_card(
+        self,
+        fields=None,
+        primary_field: str = None,
```

Comment:
as a note, only mutable fields need to be default `None`. This is `list` or `dict`